lib/elixir/lib/module/locals_tracker.ex

```diff
@@ -178,6 +189,17 @@ defmodule Module.LocalsTracker do
     end)
   end
 
+  defp undefined_from(bag, pair, meta, undefined) do
```

Comment:
This code is similar to the `reachable_from` and maybe they can be merged to run faster. Although, I'm not sure if it's better to keep them isolated to be easier to reason about it